### DISCUSSION TITLE
Add support for Simplified Chinese voice 添加简体中文配音支持

### DISCUSSION THREAD

**Discussion by Stars-Azusa** [lonqie/SchaleDB#91] [Ideas and Suggestions]
Please add options to review Simplified Chinese voices for characters, files can be extracted from the CN Server official apk.
Also, CN Server provided additional voices for Chinese New Year and the Mid-Autumn Festival, and they should be added to the voice menu.
请添加选项来预览学生的简体中文配音，语音文件可以从国服官方的apk文件中提取。
另外，国服特供了春节和中秋节的配音，它们应该被加入语音菜单中。
![b2a9ea86fc70b012f8beb51a8460c8ca](https://github.com/user-attachments/assets/df35d59a-557c-46b9-9e2a-9ad8fb9c8c66)

**Comment by Stars-Azusa** [comment]
First of all, I would like to thank the developer for providing such a convenient tool. This thing must be a lot of work, thank you for your hard work.
首先感谢开发者提供了那么便利的工具。这个东西工作量可能挺大的，辛苦了。

**Comment by lonqie** [comment]
Hi, thanks for the suggestion.

Unfortunately I don't have any plans to provide voice lines from languages on other servers such as Chinese or Korean. This is for maintenance workload and hosting reasons.

**Reply by Stars-Azusa** [reply]
Ok. Thank you.
